changelog.d/20231012_123219_andrey_x_frame_options.md

```diff
@@ -0,0 +1,4 @@
+### Security
+
+- Added X-Frame-Options: deny
```

Comment:
@azhavoro , Could you please explain why we need it?

Reply:
The X-Frame-Options HTTP response header is used to tell a browser if it should allow a web page to be displayed in a <frame>, <iframe>, or <object> HTML element. This can be useful for protecting against clickjacking attacks, which are a type of attack where an attacker tricks a user into clicking a link or button on a page that appears to be from a trusted site, but actually takes the user to a malicious site.